### ISSUE TITLE
Searches fail

### ISSUE THREAD

**Issue by Effingood** [evilhero/mylar#948]
Forgive me, I'm terrible with setting up/running python so I figure I've messed something up but haven't seen a solution on here.  

I've performed one successful search in Mylar, and since then ever search returns with:

500 Internal Server Error

The server encountered an unexpected condition which prevented it from fulfilling the request.

Traceback (most recent call last):
  File "C:\Mylar\cherrypy\_cprequest.py", line 645, in respond
    response.body = self.handler()
  File "C:\Mylar\cherrypy\lib\encoding.py", line 188, in __call__
    self.body = self.oldhandler(*args, **kwargs)
  File "C:\Mylar\cherrypy\_cpdispatch.py", line 29, in __call__
    return self.callable(*self.args, **self.kwargs)
  File "C:\Mylar\mylar\webserve.py", line 200, in searchit
    searchresults, explicit = mb.findComic(name, mode, issue=None, explicit=explicit)
  File "C:\Mylar\mylar\mb.py", line 111, in findComic
    searched = pullsearch(comicapi,comicquery,0,explicit,type)
  File "C:\Mylar\mylar\mb.py", line 63, in pullsearch
    dom = parseString(data)
  File "C:\Python27\Lib\xml\dom\minidom.py", line 1928, in parseString
    return expatbuilder.parseString(string)
  File "C:\Python27\Lib\xml\dom\expatbuilder.py", line 940, in parseString
    return builder.parseString(string)
  File "C:\Python27\Lib\xml\dom\expatbuilder.py", line 223, in parseString
    parser.Parse(string, True)
ExpatError: mismatched tag: line 1, column 316

Any ideas on how I fix this?

**Comment by evilhero** [comment]
That means there's an error in the data received from comicvine. Can you find out which series it was trying to get information for from the mylar.log file? 

What exactly were you trying to do (it looks like a Search for adding a new series). 

And do you have u our own Comicvine API key specified in the configuration?

**Comment by p3zip** [comment]
I'm having the same problem, but I think it may be due to ComicVine having troubles.  It seems like www.stage.comicvine.com is down.

**Comment by evilhero** [comment]
That would've been my next suggestion ;) 

I know comicvine was down for abit earlier, but it's back online now - it seems the api is also back, although I didn't really test it before so I can't speak about that error. 

Can you guys confirm it's working now with no errors?

**Comment by Effingood** [comment]
So I just set it up and was trying to do a search for adding a new series.  I tried it once with just searching Deadpool and once searching Ms. Marvel to test it out.  All but one attempt returned an error message like that.

I do have my own comicvine API key.  However, I've just now restarted my computer again (and ComicVine is back up, didn't realize it was down) and so far all the searches are returning results.  Thanks for your attention, evilhero and p3zip.
